### ISSUE TITLE
Filter Search - Visible Label for Input

### ISSUE THREAD

**Issue by victor-fancelli-capdevila** [orc-hfg/schaufenster#186]
According to accessibility guidelines, [Form elements should have a visible text label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Text_labels_and_names#form_elements_should_have_a_visible_text_label) (not a placeholder).

I crated a label text, and now it's like that:

![Image](https://github.com/user-attachments/assets/b4daeefa-3c72-43e8-ba57-0e6337b3845e)

Yet, it's not really nice.... 
Maybe some CSS (at base.css) may help us?

```
label #filter_text_input_label {

}
```
